code/Data exploration.R

```diff
@@ -78,5 +117,49 @@ data <- data %>%mutate(WEEKDAYWEEKEND = case_when(
   DaysOfWeek %in%  6:7  ~ "WEEKEND"
 ))
 
+##Create DATE and MONTHDATE field for joining/plotting
+data$DATE <- as.Date(substr(data$DATETIME,1,10))
+data$YEARMONTH<- as.Date(paste(substr(data$DATE,1,7), "-01", sep=""))
+
+##Join solar and rainfall
+rainfall_nsw <- rainfall_nsw %>% 
+  unite(DATE, Year, Month, Day, sep="-") %>%
```

Comment:
this date would still be a string type, I would parse this to an `as.Date()` function to ensure its a date type

Reply:
nvm I see you've done it below